assignments/objects.js

```diff
@@ -49,16 +101,30 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function() {
+    console.log(`My name is` + parent.name)
+  },
+  child: {
+    name: "George",
+    age: 50,
+    grandchild: {
+      name: "Sam",
+      age: 30
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log(parent.name);
 // Log the child's age
-
+console.log(parent.child.age);
 // Log the name and age of the grandchild
-
+console.log(parent.child.grandchild.name);
 // Have the parent speak
-
+console.log(parent.speak);
```

Comment:
```suggestion
console.log(parent.speak());
```
You need to invoke the function to be able to use it